### PR TITLE
Update portfolio

### PR DESCRIPTION
**Pull request by jelneo**

### CODE REVIEW

File: docs/team/jelenaneo.adoc [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+---
+
+=== Enhancement Proposed: Add command `remark`
+
+{Explain similar to the Undo/Redo feature above.}
+
+=== Other contributions
+
+* Nag about code quality
```
Comment (khooroko) [inline docs/team/jelenaneo.adoc:113]: nice

File: docs/team/jelenaneo.adoc [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+* Updated the GUI color scheme (Pull requests https://github.com[#33], https://github.com[#34])
+* Wrote additional tests to increase coverage from 88% to 92% (Pull requests https://github.com[#36], https://github.com[#38])
```
Comment (khooroko) [inline docs/team/jelenaneo.adoc]: remove these fake contributions please

**Review by khooroko — APPROVED** [review]

**Review by lawwman — APPROVED** [review]
Approved

File: docs/team/jelenaneo.adoc (hunk 1 above)
Comment (jaivigneshvenugopal) [reply docs/team/jelenaneo.adoc:113]: nice

**Review by jaivigneshvenugopal — APPROVED** [review]
Approved!

**Pull request merged**